Wikipedia/Code/WMFArticleViewController.m

```diff
@@ -463,6 +490,21 @@ - (UIBarButtonItem*)shareToolbarItem {
     return _shareToolbarItem;
 }
 
+- (UIBarButtonItem*)findInPageToolbarItem {
+    if (!_findInPageToolbarItem) {
+        @weakify(self);
+        _findInPageToolbarItem = [[UIBarButtonItem alloc] bk_initWithImage:[UIImage imageNamed:@"find-in-page"]
+                                                                   style:UIBarButtonItemStylePlain
+                                                                 handler:^(id sender){
+                                                                     @strongify(self);
+                                                                     if ([self canFindInPage]) { // Needed so you can't tap find icon when text size adjuster is onscreen.
```

Comment:
If its not already done, you can set a property on the popover to not allow taps to the interface while it is visible. That way you do not have to do this check.